xgcm/test/test_metrics.py

```diff
@@ -178,7 +202,7 @@ def test_get_metric_with_conditions_03():
     metric_var_2 = ds.dz_t
```

Comment:
I wonder if you should assign all metrics like this `grid = Grid(ds, coords=coords, metrics=metrics)`. This makes the code smaller and also ensures that no wrong metrics are picked even if they are available. I believe this should be changed in all of the tests. Pretty sure that wont break anything, either.